packages/twenty-front/src/modules/settings/security/components/SettingsSSOIdentitiesProvidersListCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { useNavigate } from 'react-router-dom';
+
+import { getSettingsPagePath } from '@/settings/utils/getSettingsPagePath';
+import { SettingsPath } from '@/types/SettingsPath';
+
+import { SettingsListCard } from '../../components/SettingsListCard';
+import { SettingsSSOIdentityProviderRowRightContainer } from '@/settings/security/components/SettingsSSOIdentityProviderRowRightContainer';
+import { useRecoilState } from 'recoil';
+import { SSOIdentitiesProvidersState } from '@/settings/security/states/SSOIdentitiesProviders.state';
+import { SettingsSSOIdentitiesProvidersListEmptyStateCard } from '@/settings/security/components/SettingsSSOIdentitiesProvidersListEmptyStateCard';
+import { guessIconByUrl } from '../utils/guessIconByUrl';
+import { useListSsoIdentityProvidersByWorkspaceIdQuery } from '~/generated/graphql';
+import { SnackBarVariant } from '@/ui/feedback/snack-bar-manager/components/SnackBar';
+import { useSnackBar } from '@/ui/feedback/snack-bar-manager/hooks/useSnackBar';
+
+export const SettingsSSOIdentitiesProvidersListCard = () => {
+  const navigate = useNavigate();
+  const { enqueueSnackBar } = useSnackBar();
+
+  const [SSOIdentitiesProviders, setSSOIdentitiesProviders] = useRecoilState(
+    SSOIdentitiesProvidersState,
+  );
+
+  const { loading } = useListSsoIdentityProvidersByWorkspaceIdQuery({
+    onCompleted: (data) => {
+      setSSOIdentitiesProviders(
+        data?.listSSOIdentityProvidersByWorkspaceId ?? [],
+      );
+    },
+    onError: (error: Error) => {
+      enqueueSnackBar(error.message, {
+        variant: SnackBarVariant.Error,
+      });
+    },
+  });
+
+  return !SSOIdentitiesProviders.length && !loading ? (
+    <SettingsSSOIdentitiesProvidersListEmptyStateCard />
+  ) : (
+    <SettingsListCard
+      items={SSOIdentitiesProviders}
```

Comment:
I'm a bit surprise that the SSOIdentitiesProviders match the UI items type, good coincidence!